src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstanceFactory.cs

```diff
@@ -45,28 +46,6 @@ static VisualStudioInstanceFactory()
         public VisualStudioInstanceFactory()
         {
             AppDomain.CurrentDomain.AssemblyResolve += AssemblyResolveHandler;
-            AppDomain.CurrentDomain.FirstChanceException += FirstChanceExceptionHandler;
-        }
-
-        private static void FirstChanceExceptionHandler(object sender, FirstChanceExceptionEventArgs eventArgs)
```

Comment:
@jmarolf I have full confidence that everybody is copying/pasting from a previous test anyways. The impact is minimal.

The reason xunit requires it this way is because of parallel tests -- there's no safe way to do this with static state. And the pattern is really nice IMHO.